docs/functionaldatatypes/validation.md

```diff
@@ -0,0 +1,286 @@
+---
+id: validation
+title: "Validation"
+---
+
+A `Validation[E, A]` is a data type that is either a `Success` with a value of type `A` or a `Failure` with one or more errors of type `E`.
+
+You can think of a `Validation` as looking something like this:
+
+```scala mdoc
+import zio.NonEmptyChunk
+
+sealed trait Validation[+E, +A]
+
+object Validation {
+  case class Success[+A](value: A) extends Validation[Nothing, A]
+  case class Failure[+E](errors: NonEmptyChunk[E]) extends Validation[E, Nothing]
+}
+```
+
+Thus, a `Validation` looks very much like an `Either` except that it can contain multiple errors instead of just one.
+
+The distinguishing feature of `Validation` is that it can accumulate multiple errors. This makes it extremely convenient for problems like data validation where we want to accumulate all validation errors and report them back instead of failing immediately on the first validation error.
+
+To see this, let's consider a simple example of validating a `Person` data type.
+
+```scala mdoc
+case class Person(name: String, age: Int)
```

Comment:
wish list: reuse same example in `SmartType` section where types for `name` and `age` are `SmartType`. That would show combination of Prelude features together as you did in a Zymposium talk.